using_images/db_images/mysql.adoc

```diff
@@ -68,20 +74,20 @@ running container (for debugging), you must invoke it using bash, to make sure
 the MySQL collection is enabled, for example:
 
 ----
-$ docker exec -ti CONTAINER /bin/bash -c mysql
+$ oc exec -ti -p <POD> -c <CONTAINER> /bin/bash -c mysql
 ----
 
 To enter a container from the host:
 
 ----
-$ docker exec -it <CONTAINER_ID> /bin/bash
+$ oc exec -ti -p <POD> -c <CONTAINER> /bin/bash
 ----
 
 When you enter the container, the required software collection is automatically enabled.
 
 === Environment variables
 
-The MySQL username, password, and database name can be configured with the following environment variables:
+The MySQL username, password, and database name need to be configured with the following environment variables:
```

Comment:
we should be even more explicit here about these being mandatory (except for root password) or the container will not start.